spec/features/comment_spec.rb

```diff
@@ -0,0 +1,31 @@
+feature 'adding comments' do
+  include SessionHelpers
+
+  let(:peep_content) { "Nel mezzo del cammin di nostra vita
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.